package.json

```diff
@@ -37,8 +37,11 @@
   "dependencies": {
     "aws-sdk": "^2.337.0",
     "chai": "^4.2.0",
+    "date-fns": "^1.29.0",
+    "fs": "0.0.1-security",
```

Comment:
This is normally a node built in. The addition of the word 'security' and a low version number '0.0.1' is a little ... unusual ... . I'd remove it - and let it up to a build system to polyfill it in.

Reply:
done. removed it